nautobot_floor_plan/svg.py

```diff
@@ -81,6 +81,18 @@ def _setup_drawing(self, width, depth):
 
         return drawing
 
+    @staticmethod
+    def _col_num_to_letter(col_num):
```

Comment:
Sure, it's still in draft, in other words in conceptual phase.
That may land in helpers.py as it will need more functions to convert letters into numbers and back.